app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md

```diff
@@ -8,7 +8,7 @@ related_content:
     Your teacher training personal statement : "/how-to-apply-for-teacher-training/teacher-training-personal-statement"
     Your teacher training references : "/how-to-apply-for-teacher-training/teacher-training-references"
 navigation: 30.30
-navigation_title: If your application is unsuccessful
+navigation_title: If your teacher training application is unsuccessful
```

Comment:
In the other H1 the 'you' has been changed to 'a' (Prepare for a teaching training interview) and all the 'your' have been removed from the other tiles. So I just wondered if it might be more consistent to say Unsuccessful teacher training application?